### PR TITLE
refactor(orc8r): Segregation of AnalyticsController servicer as internal

### PR DESCRIPTION
**Pull request by sabithaatla**
Signed-off-by: sabithaatla <sabitha.atla@wavelabs.ai>

## Summary

This PR deals with the segregation of Collector's internal gRPC servicers implementation.
These changes are part of the ongoing work which details the distinction between Orc8r-internal gRPC endpoints vs. external, gateway-oriented gRPC endpoints.
Moved AnalyticsCollector implementation from orc8r/cloud/go/services/analytics/servicers/ to orc8r/cloud/go/services/analytics/servicers/protected
Verified existing UT cases.

This PR was done as part of discussion by @hcgatewood [Remove gateway access to Orc8r-internal endpoints.pdf](https://drive.google.com/file/d/1NO6Qd6rU80xPh0Sb0mKWlSfDt0JbeVIT/view)

## Test Plan

All UT's passed

### CODE REVIEW

**Comment by sabithaatla** [comment]
@hcgatewood  Please review the PR

**Review by hcgatewood — APPROVED** [review]

**Comment by sabithaatla** [comment]
@hcgatewood Requesting merge

**Comment by hcgatewood** [comment]
> @hcgatewood Requesting merge

Looks like need a review from @magma/approvers-cwf

**Review by themarwhal — APPROVED** [review]

**Comment by sabithaatla** [comment]
@hcgatewood Requesting Merge

**Comment by sabithaatla** [comment]
@uri200 Requesting to review

**Review by uri200 — APPROVED** [review]

**Comment by uri200** [comment]
Is this ready to merge?

**Comment by sabithaatla** [comment]
> Is this ready to merge?

Yes

**Pull request merged**
